README.md

```diff
@@ -177,8 +183,8 @@ less: {
     options: {
       paths: ["assets/css"],
       plugins: [
-        new (require('less-plugin-autoprefix'))({browsers: ["last 2 versions"]}),
-        new (require('less-plugin-clean-css'))(cleanCssOptions)
+        new require('less-plugin-autoprefix')({browsers: ["last 2 versions"]}),
```

Comment:
Did so, however it looks like this change will remain, because while the files were changed in docs/less-examples.md the README.md doesn't look like it has been compiled since then ( also the changes removing sourceMapFileInline will now be reflected here for the same reason. )